tests/functional/test_user_restricted.py

```diff
@@ -0,0 +1,343 @@
+import pytest_bdd
+from pytest_bdd import given, then, parsers
+from functools import partial
+from .data.patients import Patient
+from .data import searches
+from .data.searches import Search
+from .data.updates import Update
+from .utils.helpers import get_role_id_from_user_info_endpoint
+import pytest
+from pytest_check import check
+from .configuration.config import BASE_URL, PDS_BASE_PATH
+from requests import Response
+import re
+import uuid
+from jsonpath_rw import parse
+
+
+AUTH_HEALTHCARE_WORKER = {
+        "api_name": "personal-demographics-service",
+        "access": "healthcare_worker",
+        "level": "aal3",
+        "login_form": {"username": "656005750104"},
+        "force_new_token": True
+    }
+
+retrieve_scenario = partial(pytest_bdd.scenario, './features/healthcare_worker_retrieve.feature')
+search_scenario = partial(pytest_bdd.scenario, './features/healthcare_worker_search.feature')
+update_scenario = partial(pytest_bdd.scenario, './features/healthcare_worker_update.feature')
+
+related_person_scenario = partial(pytest_bdd.scenario, './features/related_person.feature')
+
+status_scenario = partial(pytest_bdd.scenario, './features/status_endpoints.feature')
+
```

Comment:
is it worth covering sensitive person scenario and search returns multiple matches?

Reply:
There is a test around a sensitive person and multiple matches. Not sure what you mean?

Reply:
Also, it's worth noting, these tests should _not_ test Spine behaviour/

Reply:
spotted it now - disregard.